Cargo.toml

```diff
@@ -7,6 +7,9 @@ publish = false
 
 [dependencies]
 anyhow = "1.0"
+comfy-table = "7.0.1"
+clap = { version = "4.3.17", features = ["derive"] }
 tokio = { version = "1.29.1", features = ["full"] }
 # Using feature `unstable` for `workspaces::compile_project`.
+rust_decimal = "1.30.0"
```

Comment:
Looks like this should go on the line above to not separate the comment about workspaces unstable from the workspaces dependency.